src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLORS_AMOUNT = Color.values().length;
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        Color color = Color.values()[random.nextInt(COLORS_AMOUNT)];
+        return color.toString();
```

Comment:
Instead of using `toString()`, you should use `name()` for getting the String representation of enum constants to avoid issues if `toString()` is overridden in the enum. - `return color.name();`.